tests/integration/navigation/event/event_core.js

```diff
@@ -105,6 +105,41 @@ $.testHelper.setPushState();
 				}
 			]);
 		});
+
+		// relies on having the early popstate handler defined in early_popstate_handler.js
+		asyncTest( "Default-prevented popstate does not trigger a navigate event",
+			function( assert ) {
+				var eventNs = ".defaultPreventedPopstate";
+
+				assert.expect( 2 );
+
+				$.testHelper.detailedEventCascade( [
+					function() {
+						window.history.replaceState( { foo: "bar" }, document.title,
```

Comment:
Arguments should each be on their own line if they don't all fit on a single line.